src/scenes/Home/TerrawareHomeView/StatsCardItem.tsx

```diff
@@ -20,6 +23,8 @@ export const StatsCardItem = ({
   linkText,
   showBorder = true,
   showLink = true,
+  showTooltip = false,
```

Comment:
We could have just one attribute, so if it has tooltipText we show the tooltip and if not, we don't, but this is fine too!

Reply:
I considered it, but opted for following the pattern that the link text/enable already follows.